src/components/PharmacyCard/PharmacyCard.tsx

```diff
@@ -1,9 +1,33 @@
 import React, { useState } from "react";
-import { Card, Banner, TextContainer, Stack } from "@shopify/polaris";
+import { format } from "date-fns-tz";
+import {
+  Card,
+  Banner,
+  TextContainer,
+  Stack,
+  DataTable,
+  Button,
+} from "@shopify/polaris";
 import { DomainsMajor, LocationMajor } from "@shopify/polaris-icons";
 import Iframe from "react-iframe";
+import {
+  VaccineAvailabilityTimeslotRequirementExpandedResponse,
+  VaccineAvailabilityRequirementsResponse,
+} from "../../apiClient";
 import { useTranslation } from "react-i18next";
 
+interface RequirementsWithAvailabilityInterface
```

Comment:
The naming convention suggested by Microsoft is to prefix interfaces with `I` like C# - but for whatever reason that never took off, lol.